python/refactor/nn/loss/loss_criterion/__init__.py

```diff
@@ -0,0 +1,75 @@
+import torch as th
+import torch.nn.functional as functional
+
+logsigmoid = functional.logsigmoid
+softplus = functional.softplus
+sigmoid = th.sigmoid
+
+
+class BaseLoss(object):
+    def __call__(self, score, label):
+        raise NotImplementedError
+
+
+class HingeLoss(BaseLoss):
```

Comment:
why do we need to implement these loss functions? does pytorch not have them?

Reply:
This is something of old version, we can remove it if we want.